roles/sfia/software_engineer_2.md

```diff
@@ -46,7 +46,7 @@ Below are examples of behaviours and responsibilities a person in this role migh
 - Provides guidance and support to more junior members of the team in helping them identify what work to pick up
 - Able to iteratively deliver features of reasonable complexity, in known contexts
 - Able to architect and deliver simple features in new contexts
-- Uses a range of techniques and best practices to build easy to maintain solutions
+- Uses a range of techniques and best practises to build easy to maintain solutions
```

Comment:
Previous spelling was correct. "Practise" might be American spelling

Reply:
Yes it was the American spelling- so its incorrect 😜